dbt_subprojects/solana/models/_sector/dex/raydium_launchlab/raydium_launchlab_v1_trades.sql

```diff
@@ -0,0 +1,52 @@
+{{
+  config(
+        schema = 'raydium_launchlab_v1',
+        alias = 'trades',
+        materialized = 'view',
+        post_hook='{{ expose_spells(\'["solana"]\',
+                                    "project",
+                                    "raydium_launchlab_v1",
+                                    \'["krishhh"]\') }}')
+}}
+
+select
+      blockchain
+      , project
+      , version
+      , block_month
+      , block_date
+      , block_time
+      , block_slot
+      , trade_source
+      , token_bought_symbol
+      , token_sold_symbol
+      , token_pair
+      , token_bought_amount
+      , token_sold_amount
+      , token_bought_amount_raw
+      , token_sold_amount_raw
+      , amount_usd
+      , fee_tier
+      , fee_usd
+      , token_bought_mint_address
+      , token_sold_mint_address
+      , token_bought_vault
+      , token_sold_vault
+      , project_program_id
+      , project_main_id
+      , trader_id
+      , tx_id
+      , outer_instruction_index
+      , inner_instruction_index
+      , tx_index
```

Comment:
i would add table alias prefix on all columns for both readability and avoiding failures on ambiguous columns. same applies to filters below